src/convnets/resnets/core.jl

```diff
@@ -13,23 +13,29 @@ used to build the block for the model, see [`Metalhead.basicblock_builder`](@ref
 
 # Arguments
 
-  - `inplanes`: number of input feature maps
-  - `planes`: number of feature maps for the block
-  - `stride`: the stride of the block
-  - `reduction_factor`: the factor by which the input feature maps are reduced before
-    the first convolution.
-  - `activation`: the activation function to use.
-  - `norm_layer`: the normalization layer to use.
-  - `revnorm`: set to `true` to place the normalisation layer before the convolution
-  - `drop_block`: the drop block layer
-  - `drop_path`: the drop path layer
-  - `attn_fn`: the attention function to use. See [`squeeze_excite`](@ref) for an example.
+- `inplanes`: number of input feature maps
+- `planes`: number of feature maps for the block
+- `stride`: the stride of the block
+- `reduction_factor`: the factor by which the input feature maps are reduced before
+the first convolution.
+- `activation`: the activation function to use.
+- `norm_layer`: the normalization layer to use.
+- `revnorm`: set to `true` to place the normalisation layer before the convolution
+- `drop_block`: the drop block layer
+- `drop_path`: the drop path layer
+- `attn_fn`: the attention function to use. See [`squeeze_excite`](@ref) for an example.
```

Comment:
I don't think we should have arbitrary formatting changes in a PR like this. If it's deleting a blank line here or there, it's okay but this much. Bigger changes should be done consistently across the codebase and in their own PR.